instrumentation/opentelemetry-instrumentation-flask/tests/test_programmatic.py

```diff
@@ -570,6 +573,7 @@ def test_basic_metric_success(self):
         self.client.get("/hello/756")
         expected_duration_attributes = {
             "http.method": "GET",
+            "http.route": "/hello/<int:helloid>",
```

Comment:
You can drop this too, since the test if for old semconv.

Reply:
I think rather than dropping it should be updated:

```suggestion
            "http.target": "/hello/<int:helloid>",
```

Reply:
I'm also okay with that; I would rename the PR then and the changelog entry to make it explicit that we are adding those two attributes instead of only `http.route`

Reply:
Just to understand, the idea is to add `http.target` and `http.route` in this PR?

Reply:
Yes. It's the same attribute but with different names depending on the schema version. See https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1457#issuecomment-2229266571 and after.